test/init_test.go

```diff
@@ -84,14 +84,16 @@ func tearDown(t *testing.T, cs *clients, namespace string) {
 	}
 }
 
-func initializeLogsAndMetrics(t *testing.T) {
-	flag.Parse()
-	flag.Set("alsologtostderr", "true")
-	logging.InitializeLogger(knativetest.Flags.LogVerbose)
+func initializeLogsAndMetrics(_ *testing.T) {
```

Comment:
So I'm a little confused - where's the `t` for `t.Name()` coming from now?

Reply:
:scream: :man_facepalming: 
You are damn right @abayer :joy_cat: